customer-managed/aws/terraform/iam_redpanda_agent.tf

```diff
@@ -287,6 +288,7 @@ data "aws_iam_policy_document" "redpanda_agent1" {
     ]
     resources = [
       "arn:aws:eks:*:${local.aws_account_id}:cluster/redpanda-*",
+      "arn:aws:eks:*:${local.aws_account_id}:nodegroup/redpanda-*",
```

Comment:
question: Do we actually have nodegroups? I just looked at an arbitrary gen2 cluster in integration and got this:
```
$ aws eks list-nodegroups \
  --cluster-name redpanda-d18pslht4jfh5uobn1kg
{
    "nodegroups": []
}
```